recipes/biogeme-optimization/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "biogeme-optimization" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme-optimization-{{ version }}.tar.gz
+  sha256: ad195d3ad6841d99ad8c53b118468ad7553b9a31d95d80b26614a87644c7d8d9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.24.2
+    - scipy >=1.10.1
+    - matplotlib-base >=3.8.2
+
+test:
+  imports:
+    - biogeme_optimization
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://biogeme.epfl.ch
+  summary: Various optimization algorithms for teaching and research
+  license: MIT
+  license_family: MIT
```

Comment:
Remove these two license lines because the license for these pacakges is not the MIT license, nor it is clear it is in the MIT family.